### PR TITLE
Fixes issue 718

### LINKED ISSUE #718: Screen goes black on clicking Contents if there are no documents in Curriculum of a course
**Describe the bug**

If there are 0 documents in a course curriculum and the contents button is clicked the screen colour changes to light gray, click it again, changes to dark grey and one or two more clicks it finally changes to black.

**To Reproduce**

1. Go to Dashboard
2. Click on **Continue Learning** if you are enrolled in any course.
3. Click on **Curriculum**.
4. Click on **Documents**.
5. Click on **Contents** if there are 0 documents in the course(for example *Kotlin for Beginners* 
    has 0 documents)
6. Click on **Contents** few more times

**Logs**

No logs

**Screenshots**

Screenshot_20200515-180532_(Debug)Coding Blocks

Screenshot_20200515-180536_(Debug)Coding Blocks

Screenshot_20200515-180549_(Debug)Coding Blocks
The screen hangs when the user clicks back button.

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Samsung Galaxy M30s|
|Android Version|Android 10.0|

**Additional context**

And if user clicks the back button of phone, the screen hangs, user need to reopen the app then.

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by AkshobhyaP**
Fixes #718 

Hides FAB and shows a no content message when there is no content to display.

Screenshots for the change:

### CODE REVIEW

**Comment by AkshobhyaP** [comment]
@aggarwalpulkit596 please review.

File: app/src/main/res/layout/fragment_course_content.xml [hunk 1]
```diff
@@ -117,13 +117,32 @@
...
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/rvExpendableView"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/margin_extra_small"
-        android:overScrollMode="never"
-        tools:listitem="@layout/item_section" />
+
+   <androidx.coordinatorlayout.widget.CoordinatorLayout
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/fragment_course_content.xml]: Why do you need a coordinator layout ?

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_course_content.xml (hunk 1 above)
Comment (AkshobhyaP) [reply app/src/main/res/layout/fragment_course_content.xml]: To align the text view in the middle of the screen when the recycler view is gone so both the layouts overlap each other in the coordinator layout. In using any other layout when the recycler view was made invisible the text would jump to top as the main layout is linear.

Comment (aggarwalpulkit596) [reply app/src/main/res/layout/fragment_course_content.xml]: use match parent as height and use gravity for text view I think that would do the job also try to align the theme of text with the app

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by aggarwalpulkit596** [comment]
@AkshobhyaP any updates on this ?

**Comment by AkshobhyaP** [comment]
@aggarwalpulkit596 yeah, I will add commits by today.

**Comment by aggarwalpulkit596** [comment]
@AkshobhyaP can you add a screenshot of updated UI

**Review by aggarwalpulkit596 — APPROVED** [review]

**Comment by AkshobhyaP** [comment]
@aggarwalpulkit596 I've added the updated gif.

**Pull request merged**